### DISCUSSION TITLE
Release 1.1.0

### DISCUSSION THREAD

**Discussion by apohl79** [apohl79/audiogridder#335] [General]
Hey all,

release 1.1.0 is out now. Thanks everyone for helping us through the beta phase by reporting bugs and helping to reproduce and fix them! Please check out the [releases page](https://audiogridder.com/latest-releases/) for more info!

Cheers,
Andreas

**Comment by kastorjunior** [comment]
Bravo!

**Reply by kastorjunior** [reply]
Another 10 euros for the project from Franck...
